### PR TITLE
Upgrade repo-tools and regenerate scaffolding.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014-2017 Google Inc.
+   Copyright [yyyy] [name of copyright owner]
```
Comment (stephenplusplus) [inline LICENSE]: Oops.

Comment (jmdobry) [reply LICENSE]: No, it turns out this is how the license file is supposed to appear.

Comment (stephenplusplus) [reply LICENSE]: Attribution implied by the publisher?

Comment (stephenplusplus) [reply LICENSE]: Oh, I see now. This is the bottom of the file talking about how to attribute yourself when you copy the files. *I* was copywrong.

Comment (jmdobry) [reply LICENSE]: It was news to me too. I learned about this during some discussions with the folks who own https://opensource.google.com/docs/releasing/#checklist

**Pull request merged**